backend/src/test/java/de/sambam/gamenighttracker/controller/UserControllerTest.java

```diff
@@ -189,12 +209,16 @@ public void addNewGameSessionTest() {
         //GIVEN
         String apiGameId = "123";
         GameSession sessionToAdd = GameSession.builder().sessionState("DONE").winnerPlayerId("samBAM").build();
-        HttpEntity<GameSession> requestEntitiy = new HttpEntity<>(sessionToAdd);
+
 
         //WHEN
+        String jwtToken = logintoApp();
+        HttpHeaders header = new HttpHeaders();
+        header.setBearerAuth(jwtToken);
+        HttpEntity<GameSession> entity = new HttpEntity<>(sessionToAdd, header);
         ResponseEntity<GameSession> postResponse = testRestTemplate.exchange(
                 "http://localhost:" + port + "user/game/" + apiGameId + "/gamesessions",
-                HttpMethod.POST, requestEntitiy, GameSession.class);
+                HttpMethod.POST, entity, GameSession.class);
 
         //THEN
         assertThat(postResponse.getStatusCode(), is(HttpStatus.OK));
```

Comment:
U could verify, that the session is in the db now too.

Reply:
done ^^'